Sources/NestCLI/SwiftPackageBuilder.swift

```diff
@@ -76,10 +76,11 @@ public struct SwiftPackageBuilder {
         case (.url(let url), .latestRelease):
             let assetRegistryClient = assetRegistryClientBuilder.build(for: url)
             return try? await assetRegistryClient.fetchAssets(repositoryURL: url, version: .latestRelease).tagName
-
-        case (.ssh, .latestRelease):
+        case (.url(let url), .latestAvailableRelease):
+            let assetRegistryClient = assetRegistryClientBuilder.build(for: url)
+            return try? await assetRegistryClient.fetchNonExcludedAssets(repositoryURL: url, version: version).tagName
```

Comment:
This part is not usually used, but if it is needed in the future, it is set to retrieve the latest version excluding the excluded versions.